platinum-push-messaging.html

```diff
@@ -303,7 +303,7 @@
         }).map(function(propName) {
           return JSON.stringify(propName) + ':' + JSON.stringify(options[propName]);
         });
-        return '{' + parts.join(',') + '}';
+        return encodeURIComponent('{' + parts.join(',') + '}');
```

Comment:
Whenever I see an additional encode with a matching decode, I wonder whether something's missing. Can you confirm that the decode is being done automatically or that there's an existing decode that will take care of this?